### PR TITLE
Trb 38

### PR DESCRIPTION
**Pull request by lyancsie**
Readme.md written with minimal formatting.

**Content:**

- Environment variables
- Common flyway errors with solutions
- Heroku (to be done by @bleaksmile)

### CODE REVIEW

**Review by pitner33 — APPROVED** [review]
Seems legit 👍

File: src/main/java/com/greenfox/tribes1/Resources/Gold.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.greenfox.tribes1.Resources;
+
+import lombok.Data;
+
+@Data
```
Comment (csadesz) [inline src/main/java/com/greenfox/tribes1/Resources/Gold.java]: Try not to use @Data, I know it contains everything what we need, but it's not to stable in some cases

File: src/main/java/com/greenfox/tribes1/Resources/KingdomResource.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      cascade = CascadeType.PERSIST,
+      fetch = FetchType.EAGER
+  )
+  @JoinTable(name = "kingdom_resources",
+      joinColumns = @JoinColumn(name = "kingdom_id", referencedColumnName = "id"),
+      inverseJoinColumns = @JoinColumn(name = "resources_id", referencedColumnName = "id"))
+  private Kingdom kingdom;
+  
+  KingdomResource() {
+    amount = 500L;
```
Comment (csadesz) [inline src/main/java/com/greenfox/tribes1/Resources/KingdomResource.java:32]: Than I see, why did you use default constructor 😄

**Review by csadesz — CHANGES_REQUESTED** [review]

File: src/main/java/com/greenfox/tribes1/Resources/Gold.java (hunk 1 above)
Comment (lyancsie) [reply src/main/java/com/greenfox/tribes1/Resources/Gold.java]: Okay, thanks, I won't use it

**Review by csadesz — APPROVED** [review]

**Pull request merged**